src/App/Fossa/Config/Common.hs

```diff
@@ -315,13 +316,16 @@ parseTelemetryScope = eitherReader $ \scope ->
     "full" -> Right FullTelemetry
     _ -> Left "Failed to parse telemetry scope, expected either: full or off"
 
+fossaApiKeyCmdText :: Text
+fossaApiKeyCmdText = "fossa-api-key"
```

Comment:
[nit]: Little trick to use this as `Text`/`String`, let type inference do the work.
```suggestion
fossaApiKeyCmdText :: IsString a => a
fossaApiKeyCmdText = "fossa-api-key"
```

Reply:
More info: https://hackage.haskell.org/package/base-4.16.1.0/docs/Data-String.html#t:IsString
Also https://ghc.gitlab.haskell.org/ghc/doc/users_guide/exts/overloaded_strings.html